benchopt/plotting/plot_objective_curve.py

```diff
@@ -105,7 +105,7 @@ def plot_objective_curve(df, obj_col='objective_value', plotly=False,
     else:
         plt.legend(fontsize=14)
         plt.xlabel("Time [sec]", fontsize=14)
-        plt.ylabel(y_label, fontsize=14)
+        plt.ylabel(obj_col[10:] + ': ' + y_label, fontsize=14)
```

Comment:
where does the 10 come from?

Reply:
it's because all objectives start with `objective_` : https://github.com/benchopt/benchOpt/blob/master/benchopt/base.py#L296

Reply:
in python 3.9 I'd use `removeprefix` to be clearer, but we support 3.7 and 3.8 too

Reply:
I would do something like this:

```
def remove_prefix(text, prefix):
    if text.startswith(prefix):
        return text[len(prefix):]
    return text  # or whatever
so it's explicit
```